CRM/Contact/Form/Task/PDFLetterCommon.php

```diff
@@ -215,6 +216,29 @@ public static function postProcess(&$form) {
     CRM_Utils_System::civiExit();
   }
 
+  /**
+   * Returns the filename for the pdf by striping off unwanted characters and limits the length to 200 characters.
+   *
+   * @param CRM_Core_Form $form
+   *
+   * @return string
+   *   The name of the file.
+   */
+  private static function getFileName(CRM_Core_Form $form) {
+    if (!empty($form->getVar('_submitValues')['pdf_file_name'])) {
```

Comment:
@monishdeb this line was specifically raised as a concern in the previous PR

Reply:
sorry - it wasn't Monish it was @deb1990

Reply:
@eileenmcnaughton it is fixed now.

Reply:
Yay - looks better